clients/unshaded/src/main/java/tachyon/client/AbstractTachyonFS.java

```diff
@@ -40,7 +40,7 @@ protected AbstractTachyonFS(TachyonConf tachyonConf) {
    *
    * @param path the path of the file
    * @return The unique file id. It returns -1 if the creation failed.
-   * @throws IOException If file already exists, or path is invalid.
+   * @throws IOException if the operation failed
```

Comment:
The original description was not exhaustive, which could be misleading. Several calls down the stack you can encounter the following exceptions: BlockInfoException, InvalidPathException, FileAlreadyExistException, and also FileDoesNotExistException (as loadFileInfoFromUfs is invoked after the file is created). I find it hard to maintain recursive exception description up-to-date, so I prefer to use general ones for the functions that do not produce the exceptions.